src/rabbit_channel.erl

```diff
@@ -411,6 +415,7 @@ handle_call({info, Items}, _From, State) ->
     end;
 
 handle_call(refresh_config, _From, State = #ch{virtual_host = VHost}) ->
+    % TODO: re-init interceptor_state?
```

Comment:
The comment `refresh_config` is gone.

`rabbit_channel_interceptor` can now handle the case where an interceptor plugin is unloaded while a channel is still open.